libtenzir/include/tenzir/concept/printable/tenzir/view.hpp

```diff
@@ -42,6 +44,19 @@ struct string_view_printer : printer_base<string_view_printer> {
   }
 };
 
+struct blob_view_printer : printer_base<blob_view_printer> {
+  using attribute = view<blob>;
+
+  template <class Iterator>
+  bool print(Iterator& out, const attribute& x) const {
+    // TODO: Check that this works as expected.
+    static auto escaper = detail::make_extra_print_escaper("\"");
+    static auto p = "b\"" << printers::escape(escaper) << '"';
+    return p(out, std::string_view{reinterpret_cast<const char*>(x.data()),
+                                   x.size()});
+  }
+};
```

Comment:
Just throwing in a thought to consider: most tools I know use Base64 encoding when displaying binary data in strings. (We have routines for that in the code base already.)

Reply:
When the data actually leaves our systems (JSON, YAML, ...), the current state of the PR already prints it as Base64. This is for internal use, where it makes sense to somehow communicate that this is not a string, but a blob (currently printing e.g. `b"hello\x00\xFF"`). Also, we probably want to have a literal for blobs later (we do not want automagically parse base64-valid strings as blobs), so they need a dedicated representation anyway (again, e.g. `b"hello\x00\xFF"`)

Reply:
Sorry, I missed the right commenting location for this. Glad to see you've already went down that path!

Indeed, binary literals would be a great addition (for later).